Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -14,7 +14,9 @@
       it "returns OS_VERSION and prefix when HOMEBREW_PREFIX is a custom prefix" do
         allow(Hardware::CPU).to receive(:type).and_return(:intel)
         allow(Homebrew).to receive(:default_prefix?).and_return(false)
-        expect(described_class.os_arch_prefix_ci).to include("#{OS_VERSION}, #{described_class.custom_prefix_label}")
+        arch = ", #{described_class.arch_label}" if described_class.arch_label.present?
+        expected = "#{OS_VERSION}#{arch}, #{described_class.custom_prefix_label}"
+        expect(described_class.os_arch_prefix_ci).to include(expected)
```

Comment:
I realize - this is actually a pre-existing issue about running the tests on an M1
I will split it out into a separate PR

Reply:
Split out #12518